context/app/static/js/components/entity-search/SearchWrapper/utils.js

```diff
@@ -1,22 +1,32 @@
 import { RefinementSelectFacet } from '@searchkit/sdk';
 
+function getFacetWithGroup(facetGroup = 'Additional Facets') {
```

Comment:
```suggestion
function getFacetWithGroup(facetGroup) {
```

If nothing uses the default, don't bother with it?

Naming: I tend to reserve `get*` when the value is being pulled from somewhere else... if there's nothing external, may `build*` instead? ... Particularly since this doesn't return facets: It returns functions.

Reply:
Noted, I'll make the change.

Reply:
The default will be useful later.